sklearn/neighbors/approximate.py

```diff
@@ -459,8 +469,10 @@ def radius_neighbors(self, X, radius=None, return_distance=True):
             only present if return_distance=True.
 
         ind : array, shape (n_samples,) of arrays
-            An array of arrays of indices of the approximated nearest points
-            with in the `radius` to the query in the population matrix.
+            An array of arrays of indices of the approximate nearest points
```

Comment:
This seems to be more information than necessary; also "array of array of indices" might be harder to understand than "each element is an array of indices for neighbors within ``radius`` of the respective query". "Approximate" may not be the right word for `radius_neighbors`; the approximation only means not full recall.